front-end/src/components/PostListItem.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { Card, CardBody, CardSubtitle, CardText, CardTitle } from 'reactstrap';
+import { Post } from '../types/Post';
+import PostDetailModal from './PostDetailModal';
+
+interface Props {
+  post: Post;
+}
+
+export default function PostListItem(props:Props) {
+
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const toggle = () => setIsModalOpen(!isModalOpen);
+
+  const post: Post = props.post;
+  return (
+    <div>
+      <Card>
+        {/* TODO: Post image here? */}
```

Comment:
Yea this is something we might need to discuss about. I think Hindle talked about image post being seperate? so it might complicate things but we'll see :)

Reply:
Interesting. Yeah definitely! What I was thinking is adding logic to show either the image HTML or the text HTML, depending on the content type. I might be misunderstanding the spec though, so talking about this would be good!